awswrangler/pandas/__init__.py

```diff
@@ -0,0 +1,49 @@
+"""Pandas "proxy" package."""
```

Comment:
If the customer changes the engine, this wouldn't be reimported. We also seem to be missing a unit test case that would check for this. Something like
```python
memory_format.set("modin")
assert wr.pandas.DataFrame == modin.pandas.DataFrame

memory_format.set("pandas")
assert wr.pandas.DataFrame == pandas.DataFrame
```

Reply:
Both good catches, thanks!